app/cp_nigeria/forms.py

```diff
@@ -292,9 +292,8 @@ def __init__(self, *args, **kwargs) -> None:
             self.initial["soc_min"] = round(1 - asset.soc_min, 3)
 
         self.initial["optimize_cap"] = True
-        for field, value in zip(("name", "capex_fix", "opex_var"), (self.asset_type_name, 0, 1e-9)):
-            if field != "opex_var":
-                self.fields[field].widget = forms.HiddenInput()
+        for field, value in zip(("name", "capex_fix", "opex_var"), (self.asset_type_name, 0, 1e-4)):
```

Comment:
These settings make essentially no difference in the costs. For the scenarios I tested, the OPEX costs this causes stay well below the 1000 NGN we are using anyway to round (~40 more NGN for the Ezere scenario)

Reply:
But there is a difference between 1e-9 and 1e-4 ?

Reply:
For me, this was the threshold where the battery would stop "spilling". Same for the value that is set for the busses regarding the excess. There might be other combinations that could work but this is what I arrived at by first fixing the busses and then the battery.